### PR TITLE
feat: Added Rating component

### PR DESCRIPTION
**Pull request by Devansu-Yadav**
- Added docs and code snippet for Rating component.
- Made the component fully responsive.
- Re-factored some of the code for the component.

Preview link to the component - https://deploy-preview-10--sapphire-ui.netlify.app/docs/components/rating

### CODE REVIEW

File: Docs/Components/rating.html [hunk 1]
```diff
@@ -115,29 +115,28 @@ <h3 class="heading-3 comp-heading">Dynamic Rating</h3>
...
                         <label class="fas fa-star star" for="star-1"></label>
-                    </div>                                                                                                                                                                                                                                      
+                    </div>                                                                                                                                                            
                 </div>
                 <br>
                 <br>
 
                 <div class="code-snippet">
-                    
+                    <script src="https://emgithub.com/embed.js?target=https%3A%2F%2Fgithub.com%2FDevansu-Yadav%2FSapphire%2Fblob%2FembedBranch%2Fembed.html%23L502-L513&style=github&showBorder=on&showCopy=on"></script>
```
Comment (ravikumar1002) [inline Docs/Components/rating.html:135]: In live preview your rating code snippet is wrong

Comment (ravikumar1002) [inline Docs/Components/rating.html:135]: In live preview your rating code snippet is wrong

**Review by ravikumar1002 — APPROVED** [review]

File: Docs/Components/rating.html (hunk 1 above)
Comment (Devansu-Yadav) [reply Docs/Components/rating.html:135]: Thanks for pointing this out!

**Review by samyakshah3008 — APPROVED** [review]
Looks good to merge Devanshu!

**Pull request merged**